TestProd/src/com/recapcha/servlet/VerifyRecaptcha.java

```diff
@@ -0,0 +1,84 @@
+package com.recapcha.servlet;
+
+import java.io.BufferedReader;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+
+import org.codehaus.jackson.map.ObjectMapper;
+
+
+
+
+public class VerifyRecaptcha {
+	
+	public VerifyRecaptcha() {
+	System.out.println("Verifing...");
+	
+	}
+	
+	public static final String url = "https://www.google.com/recaptcha/api/siteverify";
+	public static final String secret = "6LfWtx0UAAAAACzKwrBAFaepdto8HwzfGybiVlPg";
+	private final static String USER_AGENT = "Mozilla/5.0";
+
+	public static Boolean verify(String gRecaptchaResponse) throws IOException {
+		if (gRecaptchaResponse == null || "".equals(gRecaptchaResponse)) {
+			System.out.println("In Data:\n");
+			return false;
+		}
+
+		try{
+			
+			System.out.println("In Verify:\n");
+			URL obj = new URL(url);
+
+			HttpURLConnection con = (HttpURLConnection) obj.openConnection();
+
+			con.setRequestMethod("POST");
+			con.setRequestProperty("User-Agent", USER_AGENT);
+			con.setRequestProperty("Accept-Language", "en-US,en;q=0.5");
+
+			String postParams = "secret=" + secret + "&response=" + gRecaptchaResponse;
+
+			// Send post request
+			
+			con.setDoOutput(true);
+			DataOutputStream wr = new DataOutputStream(con.getOutputStream());
+			wr.writeBytes(postParams);
+			wr.flush();
+			wr.close();
+			int responseCode = con.getResponseCode();
+			System.out.println("\nSending 'POST' request to URL : " + url);
+			System.out.println("Post parameters : " + postParams);
+			System.out.println("Response Code : " + responseCode);
+			BufferedReader in = new BufferedReader(new InputStreamReader(con.getInputStream()));
+			String inputLine;
+			StringBuffer response = new StringBuffer();
+			while ((inputLine = in.readLine()) != null) {
+				response.append(inputLine);
+			}
+			in.close();
+			System.out.println(response.toString());
+			
+			JsonReader jsonReader = Json.createReader(new StringReader(response.toString()));
+			JsonObject jsonObject = jsonReader.readObject();
+			jsonReader.close();
+
+			return jsonObject.getBoolean("success");
+		}
+		catch(Exception e)
+		{
+			e.printStackTrace();
```

Comment:
Log the exception here if you are not throwing it and better would be to catch a more specific exception whenever possible.